packages/tablev2/Table.tsx

```diff
@@ -91,8 +101,14 @@ function HeaderCell({ config, id, update, state }: HeaderCellProps) {
     config.sorter && (() => update({ sortBy: id, order: nextDir }));
 
   return (
-    <div className={style.th} ref={cell} onClick={onClick} role="button">
-      <div className={sortClass}>{config.header}</div>
+    <div
+      className={style.headerCell}
+      id={id}
+      ref={cell}
+      onClick={onClick}
+      role="columnheader"
```

Comment:
according to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Table_Role we might want to add an `aria-sort` here, do we?